### PR TITLE
Upgrade all dependencies

### PR DESCRIPTION
**Pull request by adaviesMOJ**

### CODE REVIEW

File: src/main/resources/db/migration/V3_31__add_index_on_booking_reference_column_for_visit_notification_event_table.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+CREATE INDEX idx_visit_notification_event_booking_reference ON visit_notification_event(booking_reference);
```
Comment (psoleckimoj) [inline src/main/resources/db/migration/V3_31__add_index_on_booking_reference_column_for_visit_notification_event_table.sql:1]: Missing trailing newline

**Review by psoleckimoj — APPROVED** [review]
LGTM!

File: src/main/resources/db/migration/V3_31__add_index_on_booking_reference_column_for_visit_notification_event_table.sql (hunk 1 above)
Comment (adaviesMOJ) [reply src/main/resources/db/migration/V3_31__add_index_on_booking_reference_column_for_visit_notification_event_table.sql:1]: I've accidentally committed this! I'll revert it, this is for another PR. Thanks Paul!

**Review by psoleckimoj — APPROVED** [review]
LGTM!

**Pull request merged**